package.json

```diff
@@ -2,10 +2,13 @@
   "name": "swagger-to-graphql",
   "version": "1.3.0",
   "author": "Roman Krivtsov",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yarax/swagger-to-graphql.git"
+  },
   "bin": "./bin/swagger2graphql",
   "dependencies": {
     "babel-runtime": "^6.25.0",
-    "graphql": "^0.12.3",
```

Comment:
{
  "scripts": {
    "precompress": "{{ executes BEFORE the `compress` script }}",
    "compress": "{{ run command to compress files }}",
    "postcompress": "{{ executes AFTER `compress` script }}"
  }
}